runtests.py

```diff
@@ -65,7 +65,7 @@
     DATABASES=DATABASES,
     DATABASE_ROUTERS=['django_sharding_library.router.ShardedRouter'],
     AUTH_USER_MODEL='tests.User',
-    ROOT_URLCONF="django_sharding.urls",
+    ROOT_URLCONF="tests.urls",
```

Comment:
In Django 1.11 `ModuleNotFoundError: No module named 'django_sharding.urls'` is raised.

So add blank `./tests/urls.py`